llvm/test/tools/repo-fragments/tool-options-test.ll

```diff
@@ -0,0 +1,34 @@
+; Check repo-fragments options, which include:
+;    1) Check the default case by dumping _all_ of the names from a compilation;
```

Comment:
The emphasis on “\_all\_” isn’t necessary.

Reply:
Corrected it.